package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "cfpdx-301-portfolio",
+  "version": "1.0.0",
+  "description": "Portfolio for code works",
+  "main": "server.js",
+  "dependencies": {
+    "body-parser": "^1.17.1"
```

Comment:
You are missing express... (you've likely installed it globally, but still need to have it as a dependency for the project).